context/app/static/js/components/Detail/ProvTabs.jsx

```diff
@@ -121,6 +128,10 @@ function ProvTabs(props) {
 
 ProvTabs.propTypes = {
   uuid: PropTypes.string.isRequired,
+  assayMetadata: PropTypes.objectOf(
+    PropTypes.oneOfType([PropTypes.array, PropTypes.object, PropTypes.string, PropTypes.number]),
```

Comment:
When we can't be more precise than this, maybe just eslint ignore? If someone decided to add a boolean, is that actually a problem?